proprietary/design-tokens/build/build-tokens.mjs

```diff
@@ -0,0 +1,244 @@
+ */
+StyleDictionary.registerTransform({
+  name: 'spacing/calc',
+  type: 'value',
+  transitive: true,
+  matcher: function(token) {
+    const {type, value, isSource} = token
+    return type === 'spacing' && isSource && value.match(/[\+\-\*\/]/g);
+  },
+  transformer: function(token) {
+    return `calc(${token.original.value})`;
+  }
+})
+
+registerTransforms(StyleDictionary, {
+  /* options here if needed */
+});
+
+const cleanName = (n) => n.toLowerCase().replaceAll(/\/?(default|\s\(.+\))/g, '').replaceAll(' ', '-');
+const isFigmaToken = (name) => name.startsWith('figma');
+const excludeSystemTokens = ({ name }) => ![isFigmaToken].some((fn) => fn(name));
+const extractProductFromName = (name) => name.split('/').slice(0, -2).join('-');
+const extractModeFromName = (name) => name.split('/').slice(-2, -1).toString();
+
+/**
+ * Gets the different groups from the themes object and groups them.
+ * @param {*} themes 
+ * @returns {Object<string, object[]>} Object with groups
+ */
+const getThemeGroupObjects = (themes) => {
+  // Sort themes by groups
+  const groups = {};
+  themes.forEach(theme => {
+    if (theme.group) {
+      groups[cleanName(theme.group)] = [...(groups[cleanName(theme.group)] ?? []), theme];
+    } else {
+      throw new Error(
+        `Theme ${theme.name} does not have a group property, which is required for multi-dimensional theming.`,
+      );
+    }
+  });
+
+  return Object.entries(groups)
+    .reduce((_sortedObj, [k,v]) => ({
+      ..._sortedObj, 
+      [k]: v
+    }), {});
+};
+
+/**
+ * Copied from \@tokens-studio/sd-transforms; 
+ * @see https://github.com/tokens-studio/sd-transforms/blob/main/src/permutateThemes.ts#L55
+ * @param {object} tokensets Object containing tokensets
+ * @returns {string[]} Array containing filtered tokensets
+ */
+const filterTokenSets = (tokensets) => {
+  return (
+    Object.entries(tokensets)
+      .filter(([, val]) => val !== 'disabled')
+      // ensure source type sets are always ordered before enabled type sets
+      .sort((a, b) => {
+        if (a[1] === 'source' && b[1] === 'enabled') {
+          return -1;
+        } else if (a[1] === 'enabled' && b[1] === 'source') {
+          return 1;
+        }
+        return 0;
+      })
+      .map(entry => entry[0])
+  );
+}
+
+/**
+ * Returns Products combined with Modes and Viewports
+ * @param {string} name 
+ * @param {Array.<[mode: string, viewports: Array.<string>]>} combined Array with mode and viewports
+ * @returns {Array.<{name: string, mode: string, viewport: string}>} Array of objects with name, mode and viewport
+ */
+const getProductCombinations = (name, combined) => combined.flatMap((combo) => {
+  const [mode, viewports] = combo;
+  return viewports.map((viewport) => ({name: cleanName(name), mode, viewport}));
+});
+
+const getTokensByMode = (groups, mode) => filterTokenSets(groups['mode'].find(({name}) => name.includes(mode)).selectedTokenSets);
```

Comment:
```suggestion
const getTokensByMode = (groups, mode) => filterTokenSets(groups.mode.find(({name}) => name.includes(mode)).selectedTokenSets);
```

Is dit een overblijfsel van de "` (beta)`" suffixes? In dat geval kunnen al deze gevallen vervangen worden